bioconda_utils/lint/__init__.py

```diff
@@ -0,0 +1,446 @@
+"""Recipe Linter
+
```

Comment:
Very helpful. Reading this before even looking at the code gave me a lot of insight and I felt comfortable that I could at least figure things out. Then when reading the code everything made a lot of sense.

Reply:
Great! :)